.github/workflows/main.yml

```diff
@@ -37,15 +38,23 @@ jobs:
         run: |
           mvn -P ci -B test --settings .github/maven-settings.xml
       - name: Publish Docker image
-        if: github.ref == 'refs/heads/dev' || github.ref == 'refs/heads/master'
+        if: github.ref == 'refs/heads/dev' || github.ref == 'refs/heads/master' || github.ref == env.Q1_TEST_BRANCH
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
           mvn -P ci -B package --settings .github/maven-settings.xml -DskipTests
           docker login docker.pkg.github.com -u ${GITHUB_REPOSITORY} -p ${GITHUB_TOKEN}
           docker build --tag ${IMAGE} .
           docker push ${IMAGE}
-
+      - name: Deploy to Q1 (Test branch)
+        uses: nais/deploy/actions/deploy@v1
+        if: github.ref == env.Q1_TEST_BRANCH
+        env:
+          APIKEY: ${{ secrets.NAIS_DEPLOY_APIKEY }}
+          PRINT_PAYLOAD: true
+          CLUSTER: dev-fss
+          RESOURCE: .nais/nais-q1.yml
+          VAR: version=${{ github.sha }}
```

Comment:
Skal dette være med inn i prod? Det har kanskje ikke noe å si?

Reply:
Jupp den er med der, og brukes til å sette miljøvariablen `APP_VERSION` for appen.
Mulig vi ikke trenger den noe mer, men ble tidligere brukt for på selftest-siden for å si noe om hvilken versjon som var aktiv.